src/core/lib/security/credentials/ssl/ssl_credentials.cc

```diff
@@ -241,7 +242,7 @@ grpc_ssl_server_credentials_create_options_using_config(
   retval = options;
```

Comment:
i can't seem to add a comment at the start of function so i do it here:
it seems that you don't need `retval`, and instead can initialize `options = NULL;` at the top of the function, and `return options`